velero-plugins/common/types.go

```diff
@@ -19,6 +19,8 @@ const RestoreServerVersion string = "openshift.io/restore-server-version"
 const BackupRegistryHostname string = "openshift.io/backup-registry-hostname"
 const RestoreRegistryHostname string = "openshift.io/restore-registry-hostname"
 
+const SkipImages string = "openshift.io/skip-images"
```

Comment:
Our newer annotations tend to use "migration.openshift.io" -- should we move to that here as well?

Reply:
+1

Reply:
My thought process was to not use migration since this would be set outside of CAM/OADP context. But find to do that if you think it's better